discovery-provider/src/queries/get_attestation.py

```diff
@@ -61,10 +67,14 @@ class AttestationError(Exception):
     pass
 
 
-def is_valid_oracle(address: str):
-    # TODO: flesh this out, refresh oracle addresses [AUD-729]
-    default_oracle_address = shared_config["discprov"]["default_oracle_address"]
-    return address == default_oracle_address
+def is_valid_oracle(address: str) -> bool:
+    redis = get_redis()
+    oracle_addresses = redis_get_or_restore(redis, oracle_addresses_key)
+    if not oracle_addresses:
+        oracle_addresses = get_oracle_addresses_from_chain(redis)
+    else:
+        oracle_addresses = oracle_addresses.decode().split(",")
```

Comment:
do we need to return address in oracle_addresses? isn't it already an array?

Reply:
from the chain it's an array, but not from redis. saving it as comma-separated address string. has to do with bytes format in redis_set_and_dump and redis_get_or_restore

Reply:
gotcha. yeah i think we can just do redis.get/set